docker-bake.hcl

```
@@ -37,9 +37,7 @@ target "all-arch" {
     "linux/arm",
```

Default arm variant is v7. I think it should be distinguished (maybe include v6 as well).

```suggestion
    "linux/arm/v6",
    "linux/arm/v7",
```

---

https://github.com/containerd/containerd/blob/master/platforms/platforms.go#L100 `linux/arm` is normalized. I don't really want to add v6 very much as there is no definition of what v6 means. Afaics, it means different things in alpine vs debian images. Better to let it die. Also, not related to this pr.